docs/changelog/119826.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 119826
```

Comment:
Let's go ahead and remove this since this is a refactoring PR and isn't adding a new feature.